stories/DataTable/constants/sampleChocolates.js

```diff
@@ -0,0 +1,148 @@
+// Author https://github.com/schloerke/cranvasOLD/blob/master/files/data/chocolates.csv
```

Comment:
I know you put some work into these tasty chocolates, but you can just map some disabled props in the story book using the existing data sets.

```
import data from '../constants/desserts';

// then outside the storybook component
const customData = data.map(datum => ({ ...datum, disabled: false }));
// Then arbitratily just disable some
customData[1].disabled = true;
customData[3].disabled = true;
```